docs/demos/onvif-authentication-access.md

```diff
@@ -0,0 +1,306 @@
+
+See [the cluster setup steps](../user-guide/cluster-setup.md#configure-crictl) for information on how to set the crictl configuration variable `AKRI_HELM_CRICTL_CONFIGURATION`
+
+ ```bash
+export AKRI_HELM_CRICTL_CONFIGURATION="--set kubernetesDistro=k8s"
+```
+
+Install an Akri configuration named `akri-onvif` that uses debug echo discovery handler
+
+```bash
+helm install akri akri-helm-charts/akri-dev \
+   $AKRI_HELM_CRICTL_CONFIGURATION \
+   --set onvif.discovery.enabled=true \
+   --set onvif.configuration.name=akri-onvif \
+   --set onvif.configuration.enabled=true \
+   --set onvif.configuration.capacity=3 \
+   --set onvif.configuration.brokerPod.image.repository="nginx" \
+   --set onvif.configuration.brokerPod.image.tag="stable-alpine"
+```
+
+Here is the result of running the installation command above on a cluster with 1 control plane and 2 work nodes.  There is one Onvif camera connects to the network, thus 1 pods running on each node.
+
+```bash=
+$ kubectl get nodes,akric,akrii,pods
+NAME           STATUS   ROLES           AGE   VERSION
+node/kube-01   Ready    control-plane   22d   v1.26.1
+node/kube-02   Ready    <none>          22d   v1.26.1
+node/kube-03   Ready    <none>          22d   v1.26.1
+
+NAME                               CAPACITY   AGE
+configuration.akri.sh/akri-onvif   3          62s
+
+NAME                                 CONFIG       SHARED   NODES                   AGE
+instance.akri.sh/akri-onvif-029957   akri-onvif   true     ["kube-03","kube-02"]   48s
+
+NAME                                              READY   STATUS    RESTARTS   AGE
+pod/akri-agent-daemonset-gnwb5                    1/1     Running   0          62s
+pod/akri-agent-daemonset-zn2gb                    1/1     Running   0          62s
+pod/akri-controller-deployment-56b9796c5-wqdwr    1/1     Running   0          62s
+pod/akri-onvif-discovery-daemonset-wcp2f          1/1     Running   0          62s
+pod/akri-onvif-discovery-daemonset-xml6t          1/1     Running   0          62s
+pod/akri-webhook-configuration-75d9b95fbc-wqhgw   1/1     Running   0          62s
+pod/kube-02-akri-onvif-029957-pod                 1/1     Running   0          48s
+pod/kube-03-akri-onvif-029957-pod                 1/1     Running   0          48s
+```
+Dump the environment variables from the container to check the device uuid from the container's environment variables. Below is an example, the Onvif discovery handler discovers the camera and expose the device's uuid. **Write down the device uuid for later use**. Note that in real product scenarios, the device uuids are acquired directly from the vendors or already known before installing Akri Configuration.
+
+```bash=
+$ kubectl exec -i kube-02-akri-onvif-029957-pod -- /bin/sh -c "printenv|grep ONVIF_DEVICE"
+ONVIF_DEVICE_SERVICE_URL_029957=http://192.168.1.145:2020/onvif/device_service
+ONVIF_DEVICE_UUID_029957=3fa1fe68-b915-4053-a3e1-ac15a21f5f91
+```
```

Comment:
It is simpler to get this information from the Akri Instance properties

Reply:
get uuid from akrii properties